models/src/main/kotlin/io/provenance/api/models/cee/execute/ContractExecutionResponse.kt

```diff
@@ -1,16 +1,10 @@
 package io.provenance.api.models.cee.execute
 
-import com.fasterxml.jackson.annotation.JsonSubTypes
 import com.fasterxml.jackson.annotation.JsonTypeInfo
 import io.provenance.api.models.p8e.TxResponse
 import java.util.UUID
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
```

Comment:
Should we change the property to `@class` or will `type` automagically work here?

Reply:
I ran the unit test for this and it was fine. `@class` just seemed to be more common so i went with that for the other `@JsonTypeInfo` usage. Up to you if you want me to change it or not, dont think it will break `service-dart` or anything .